### PR TITLE
Enhancements to project summary fields

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues

Fixes cityofaustin/atd-data-tech/issues/10754

## Testing
**URL to test:** 

https://deploy-preview-909--atd-moped-main.netlify.app

**Steps to test:**
1. Go to the project summary page for a new or existing project that has a parent project.
2. Ensure that the parent project field is now aligned with the rest of the fields.
3. Update the description and website fields to have very longgggggggg strings. The strings should now break and wrap when not editing.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by roseeichelmann** [comment]
I left in the package-lock commit because it bumps the version from 1.13.0 to 1.14.0. Let me know if I should leave it out and I will revert that change

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryParentProjectLink.js [hunk 1]
```diff
@@ -21,13 +20,18 @@ const ProjectSummaryParentProjectLink = ({ data, classes }) => {
...
+        alignItems="center"
       >
         <RouterLink
           id="projectKnackSyncLink"
           className={"MuiTypography-body1"}
           to={`/moped/projects/${parentProjectId}`}
         >
-          <Typography variant={"inherit"} color={"primary"}>
+          <Typography
+            className={classes.fieldLabelText}
```
Comment (roseeichelmann) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryParentProjectLink.js:31]: this class gives us that proper alignment

**Comment by patrickm02L** [comment]
The Description and Website fields are a go. 

The alignment of the Parent project link is still off hanging over the left edge. Scoot over a few points. I think that link should have the same "underlined" hover effect as the other ones on the page. 

Screen Shot 2022-12-19 at 5 08 31 PM

**Comment by roseeichelmann** [comment]
@patrickm02L Hmm weird, it's aligned for me. Are you using the netlify link? I wonder how it looks for other people. I can definitely make it underline on hover like the other links.

image

**Review by frankhereford — APPROVED** [review]
🚢 🚢 

Looks great, works great, reads great - this is right on the money.

It also caused me to find what may be the longest, meaningfully named URL:

http://llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch.co.uk, which is pronounced like this: https://www.youtube.com/watch?v=fHxO0UdpoxM.

**Review by mddilley — APPROVED** [review]
Tested on the deploy preview and everything looked just as expected! 🚀 ✨

**Comment by patrickm02L** [comment]
The Parent project link is aligned for me now. Everything looks good.
Screen Shot 2022-12-20 at 3 34 37 PM

**Review by patrickm02L — APPROVED** [review]
Good to go! 🤘

**Comment by johnclary** [comment]
thanks, @roseeichelmann! 🚢 ✨✨✨✨✨✨✨✨✨

**Pull request merged**